imports/plugins/core/versions/server/migrations/4_update_templates_priority.js

```diff
@@ -0,0 +1,40 @@
+import { Migrations } from "/imports/plugins/core/versions";
+import { Packages } from "/lib/collections";
+
+const query = { layout: { $type: 3 } }; // gets docs with layout objects
```

Comment:
It seems this query will affect all packages with layout, not just our own packages. might need to include a list of our packages like so:

```
name: { $in: ["reaction-core", "reaction-<other_name>", ...] }
```

You probably don't need all package names, only ones with actual layout.